avaframe/com1DFA/com1DFA.py

```diff
@@ -120,7 +122,7 @@ def com1DFAMain(avalancheDir, cfgMain, cfgFile='', relThField='', variationDict=
             # +++++++++++++++++++++++++++++++++
             # ------------------------
             particlesList, fieldsList, tSave, dem, reportDict, cfgFinal, Tcpu = com1DFA.com1DFACore(cfg, avalancheDir,
-                    cuSim, inputSimFiles, outDir, relThField)
+                    cuSim, inputSimFiles, outDir, relThField=relThField)
```

Comment:
Since the adding of relThField  to add the functionality of having non uniform release heights is a big improvement, please add this as improvement in the PR description